safari/Refined GitHub.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		1F9945322A07DB5E005BC236 /* resources in Resources */ = {isa = PBXBuildFile; fileRef = 1F9945312A07DB5E005BC236 /* resources */; };
+		1F9945332A07DB5E005BC236 /* resources in Resources */ = {isa = PBXBuildFile; fileRef = 1F9945312A07DB5E005BC236 /* resources */; };
```

Comment:
Since `distribution` can't be changed, I added `distribution/resources` as a symlink:

<img width="109" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/236680072-b901eddc-ab57-4b33-b648-944ad6465be5.png">